src/index.spec.js

```diff
@@ -336,6 +336,7 @@ describe('Dropzone', () => {
       );
       dropzone.find(ChildComponent).simulate('dragEnter', { dataTransfer: { files } });
       dropzone.simulate('dragEnter', { dataTransfer: { files } });
+      dropzone.simulate('dragEnter', { dataTransfer: { files } }); // make sure we de-dupe events
```

Comment:
Hmm, not sure I get this line. You're entering 2 times and leaving just 1 but asserts are passing? They shouldn't or am I missing something?